modules/git/command.go

```diff
@@ -321,7 +331,7 @@ func (c *Command) run(ctx context.Context, skip int, opts *RunOpts) error {
 
 	startTime := time.Now()
 
-	cmd := exec.CommandContext(ctx, c.prog, c.args...)
+	cmd := exec.CommandContext(ctx, c.prog, append(append([]string{}, c.configArgs...), c.args...)...)
```

Comment:
```suggestion
	cmd := exec.CommandContext(ctx, c.prog, append(c.configArgs, c.args...)...)
```

Reply:
I can ensure `append(c.configArgs, c.args...)` will return a new slice which will not change `c.configArgs` . ref: https://go.dev/ref/spec#Appending_and_copying_slices